janitor/functions/min_max_scale.py

```diff
@@ -1,116 +1,124 @@
-import pandas_flavor as pf
-import pandas as pd
-
-from janitor.utils import deprecated_alias
-
-
-@pf.register_dataframe_method
-@deprecated_alias(col_name="column_name")
-def min_max_scale(
-    df: pd.DataFrame,
-    old_min=None,
-    old_max=None,
-    column_name=None,
-    new_min=0,
-    new_max=1,
-) -> pd.DataFrame:
-    """
-    Scales data to between a minimum and maximum value.
-
-    This method mutates the original DataFrame.
```

Comment:
The line was deleted. See [below](https://github.com/pyjanitor-devs/pyjanitor/pull/1107#discussion_r885390547)